src/main/java/ro/vladutit/Don/t/forget/v2/controller/ItemController.java

```diff
@@ -16,14 +16,14 @@
     @Autowired
     private CategoryService categoryService;
 
-    @PostMapping("/addNewItem")
+    //add new item
```

Comment:
same here

Reply:
I have made these changes everywhere. I made an update to commit (--amend). Please take a look. Thank you.